tests/python/unittest/test_link_params.py

```diff
@@ -412,17 +412,12 @@ def schedule_fn(sch):
             return True
         return False
 
-    link_params = True
-
```

Comment:
What’s the impact of the PR on this file :eyes:?

Reply:
It's per @masahi's suggestion from the previous PR: https://github.com/apache/tvm/pull/12626#discussion_r956952473

Reply:
Ooops sure!